provisioning/resources/configs/snowplow-stream-enrich.hocon

```diff
@@ -1,15 +1,10 @@
-# Copyright (c) 2013-2018 Snowplow Analytics Ltd. All rights reserved.
+# Copyright (c) 2016-present Snowplow Analytics Ltd. All rights reserved.
```

Comment:
This file is still named `stream-enrich`, where elsewhere it was renamed `enrich` - just want to validate that this is not an error.